package.json

```
@@ -31,7 +31,7 @@
     "stylelint-scss": "^3.14.0"
   },
   "peerDependencies": {
-    "stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0 || ^13.0.0",
+    "stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0",
```

```suggestion
    "stylelint": "^14.0.0",
```

I originally thought this would be a non-breaking change, but it is probably easier update to the latest version of stylelint-config-recommended in this pull request (use `github:stylelint/stylelint-recommended-config#prepare-6` for now) as that version includes a rule which we added for 14. See https://github.com/stylelint/stylelint-config-recommended/pull/123

You'll need to use `github:stylelint/stylelint#14` as a dev dependency too.